pylint/checkers/base.py

```diff
@@ -2075,6 +2147,64 @@ def _name_became_keyword_in_version(name, rules):
                 return ".".join(str(v) for v in version)
         return None
 
+    @staticmethod
+    def _assigns_typevar(node: Optional[nodes.NodeNG]) -> bool:
+        """Check if a node is assigning a TypeVar."""
+        if isinstance(node, astroid.Call):
+            inferred = utils.safe_infer(node.func)
+            if (
+                inferred
+                and isinstance(inferred, astroid.ClassDef)
+                and inferred.qname() == TYPING_TYPE_VAR_QNAME
+            ):
+                return True
+        return False
+
+    @staticmethod
+    def _check_typevar_variance(
+        name: str, node: nodes.AssignName
+    ) -> List[Tuple[str, str, str]]:
+        """Check if a TypeVar has a variance and if it's included in the name.
+
+        Returns the args for the message to be displayed.
+        """
+        args_to_display: List[Tuple[str, str, str]] = []
+
+        if isinstance(node.parent, nodes.Assign):
+            keywords = node.assign_type().value.keywords
+        elif isinstance(node.parent, nodes.Tuple):
+            keywords = (
+                node.assign_type().value.elts[node.parent.elts.index(node)].keywords
+            )
+
+        if keywords:
+            for kw in keywords:
+                if (
+                    kw.arg == "covariant"
+                    and kw.value.value
+                    and not name.endswith("_co")
+                ):
+                    suggest_name = re.sub("_contra$", "", name)
+                    suggest_name += "_co"
+                    args_to_display.append((name, "covariant", suggest_name))
+                if (
```

Comment:
I would suggest adding an empty line between these block. That should make it more readable.